hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -577,6 +578,7 @@
         regs: {hart: "0x21200000"},
         dbg:  {soc_dbg: "0x00000000"},
       },
+      cpu_not_visible: ["mem", "regs", "dbg"]
```

Comment:
It's a bit sad to have the names appearing twice. Could we switch it so that we end up with something like this?
```
      // Note that this module also contains a bus host.
      base_addrs: {
        mem:  {hart: "0x00040000", cpu_visible: false},
        regs: {hart: "0x21200000", cpu_visible: false},
        dbg:  {soc_dbg: "0x00000000", cpu_visible: false},
      },
```
and we could make it so a region has a default visibility attribute (true, presumably)

Reply:
I changed the implementation according to that. It was required to add some special cases when iterating over that struct. Should we add a top-level validation, to ensure that there is no address space named `cpu_visible`?